### ISSUE TITLE
[BUG] Accounts displayed multiply themselves in "following" and "followers" sections

### ISSUE THREAD

**Issue by retiolus** [H-PixelDroid/PixelDroid#219]
* [x] I checked the release notes for known issues
* [x] I checked the existing issues
* [x] I checked that the issue is not related to my instance configuration, and that my instance has the latest version of pixelfed

In "following" or "followers" section of an account, the accounts displayed multiply themselves (sometimes to "infinity" without stopping).

**Comment by Wv5twkFEKh54vo4tta9yu7dHa3** [comment]
Yep, I have signaled this to dansup, and it will be looked into. We're just showing what the API gives us though :shrug:
